usmqe_tests/api/gluster/conftest.py

```diff
@@ -63,24 +64,74 @@ def invalid_volume_id(request):
 
 
 @pytest.fixture
-def valid_volume_configuration(valid_volume_name):
+def valid_brick_name():
+    """
+    Generate valid brick name.
+    """
+    return pytest.config.getini("usm_brick_name")
+
+
+@pytest.fixture
+def valid_brick_path(valid_brick_name):
+    """
+    Generate valid brick path. Bricks are generated in directory:
+    ``/tendrl_gluster_bricks/<brick_name>_mount``
+
+    as described in https://github.com/Tendrl/gluster-integration/issues/320
+    and in https://github.com/nnDarshan/documentation/blob/415c3e9fd50d1a6e38ce6cc84abbde8db31475c4/
+                   gluster_brick_provisioning.adoc
+    """
+    return os.path.join(os.path.sep, "tendrl_gluster_bricks", "{}_mount".format(
+        valid_brick_name))
+
+
+@pytest.fixture
+def valid_devices(valid_session_credentials, count=1):
+    """
+    Generate device paths.
+
+    Args:
+        count (int): How many device paths should be generated.
+                     There have to be enough devices.
+    """
+
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    device_info = [list(x["localstorage"]["blockdevices"]["all"].values())[0]
+                   if x["localstorage"]["blockdevices"]["all"] is not None
+                   else None for x in api.get_nodes()["nodes"]]
+    device_info = [x for x in device_info if x is not None]
```

Comment:
why is there this second cycle?
Is it not possible to have them together, like this:
```
for x in api.get_nodes()["nodes"] if x["localstorage"]["blockdevices"]["all"] is not None
```